colcon_clean/base_handler/install.py

```diff
@@ -1,16 +1,21 @@
 # Copyright 2021 Ruffin White
 # Licensed under the Apache License, Version 2.0
 
-import os
+from pathlib import Path
+from typing import List
 
 from colcon_clean.base_handler import BaseHandlerExtensionPoint
+from colcon_core.logging import colcon_logger
+from colcon_core.package_descriptor import PackageDescriptor
 from colcon_core.plugin_system import satisfies_version
 
+logger = colcon_logger.getChild(__name__)
```

Comment:
Does this result in the same output in colcon logs for the colcon clean verb? I hadn't noticed this logger pattern when basing this extension from the other existing extensions years ago.

Reply:
I was curious in how the logs may differ with these logger improvements. I.e. running the example in the README with and without these changes and diffing the workspace `./log` base path, sans the timestamps. I'll take a peak at this aspect once there are tests I can try.